Frends.LDAP.SearchObjects/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [2.2.0] - 2025-03-17
```

Comment:
This change looks like a breaking change, so go for 3.0.0 version? Or am I mitaking here?
Also, this is a pretty big update in terms of parameter amount, so it is good to mention how to upgrade from previous version. E.g. if someone has been using the Attributes already to get only a subset of attrs from LDAP, then they have to make sure that SearchOnlySpecifiedAttributes is true and make sure that content encoding is set to XXX (I am actually unsure what value they need to use, UTF8?.. with or without BOM?).